src/main/java/core/basesyntax/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.FruitTransactionDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
```

Comment:
The class should not begin with an empty line. Please remove the empty line at the start of the class definition to adhere to the formatting guidelines.